functions/channelCapture/chatbotCallback.protected.ts

```diff
@@ -42,14 +47,15 @@ type EnvVars = AWSCredentials & {
   SURVEY_WORKFLOW_SID: string;
 };
 
-export type Body = Partial<WebhookEvent> & {};
+export type Body = Partial<ConversationWebhookEvent & ProgrammableChatWebhookEvent> & {};
```

Comment:
```suggestion
export type Body = ConversationWebhookEvent | ProgrammableChatWebhookEvent;
```
does not works here?

Reply:
Not without rewriting the validation logic, which I can't be bothered doing since Programmable Chat support will be going away